llvm/tools/llvm-gsymutil/llvm-gsymutil.cpp

```diff
@@ -177,6 +179,16 @@ static void parseArgs(int argc, char **argv) {
 
   LookupAddressesFromStdin = Args.hasArg(OPT_addresses_from_stdin);
   StoreMergedFunctionInfo = Args.hasArg(OPT_merged_functions);
+
+  for (const llvm::opt::Arg *A : Args.filtered(OPT_callsites_from_yaml_EQ))
+    if (A->getValue() && A->getValue()[0] != '\0')
+      CallSiteYamlPaths.emplace_back(A->getValue());
```

Comment:
Multiple YAML is not a common use case - I can remove it since there is no test for it. 

There is currently no validity checks for the YAML input. This means things like you can specify offsets outside the function code range, you can specify the same function multiple times, you can specify non-existent functions, etc ... and won't get errors. Multiple YAML identical YAML files would be same effect as one YAML file with duplicate entries (which currently we don't check).

Since this format is only for testing, I'm thinking it's not super critical to have strict error checking. If we want to add strict error checking, I think a separate PR would be better - for reduced complexity.